apps/web/components/ReviewsAccordion/ReviewsAccordion.vue

```diff
@@ -121,16 +129,5 @@ const deleteReview = () => {
   refreshReviews();
   emits('on-list-change');
 };
-
-watch(
-  () => reviewsOpen.value,
-  (value) => {
-    if (value) {
-      fetchProductReviews(
-        Number(productGetters.getItemId(product.value)),
-        productGetters.getVariationId(product.value),
-      );
-    }
-  },
-);
```

Comment:
This is what @doproiescu-plenty is working on. I also don't think this is correct because the collapsable should be there, just initially visible. Please keep these changes as they ware in this PR.